<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NGAuthOptions</class>
 <widget class="QWidget" name="NGAuthOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>490</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_23">
   <item>
    <widget class="QGroupBox" name="groupBox_29">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string>User info</string>
     </property>
     <widget class="QLabel" name="avatar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionText">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Not authorized</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="authGroupBox">
     <property name="title">
      <string>Authorize options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_23">
        <item row="3" column="0">
         <widget class="QLabel" name="tokenEndpointLabel">
          <property name="text">
           <string>Token endpoint</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="authEndpointEdit">
          <property name="inputMethodHints">
           <set>Qt::ImhUrlCharactersOnly</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="endpointEdit">
          <property name="inputMethodHints">
           <set>Qt::ImhUrlCharactersOnly</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="userInfoEndpoinLabel">
          <property name="text">
           <string>User info endpoint</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="userInfoEndpointEdit">
          <property name="inputMethodHints">
           <set>Qt::ImhUrlCharactersOnly</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="enpointLabel">
          <property name="text">
           <string>Endpoint</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="authEndpointLabel">
          <property name="text">
           <string>Auth endpoint</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="tokenEndpointEdit">
          <property name="inputMethodHints">
           <set>Qt::ImhUrlCharactersOnly</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="authTypeSelector">
          <item>
           <property name="text">
            <string>NextGIS ID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Keycloak</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="codeChallengeCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="codeChallengeLabel">
          <property name="text">
           <string>Use code challenge</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="availableEndpointLabel">
          <property name="text">
           <string>Not available</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="defaultsButton">
          <property name="text">
           <string>Defaults</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signinButton">
          <property name="text">
           <string>Sign in</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="sendCrashes">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Send crashes to NextGIS</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
